Source/Common/Config.cpp

```diff
@@ -108,7 +108,7 @@ class MainLoader final : public OptionMapper {
 public:
   explicit MainLoader(FEXCore::Config::LayerType Type);
   explicit MainLoader(fextl::string ConfigFile);
-  explicit MainLoader(FEXCore::Config::LayerType Type, const char* ConfigFile);
+  explicit MainLoader(FEXCore::Config::LayerType Type, const std::string_view ConfigFile);
```

Comment:
```suggestion
  explicit MainLoader(FEXCore::Config::LayerType Type, std::string_view ConfigFile);
```

Since this is just passed by value

Reply:
Good catch, fixed.